pkg/apis/certmanager/v1/types_certificate.go

```diff
@@ -165,12 +165,20 @@ type CertificateSpec struct {
 	// The `name` field in this stanza is required at all times.
 	IssuerRef cmmeta.ObjectReference `json:"issuerRef"`
 
-	// IsCA will mark this Certificate as valid for certificate signing.
+	// IsCA indicates that the certificate should be usable for certificate signing.
```

Comment:
suggestion: Would it make sense here for the language to be something like "IsCA will request that this certificate should be marked as a CA certificate" - i.e., focusing on the _request_ ?

Logic being that the Certificate resource is a friendly way to generate a CertificateRequest which is in turn a wrapper for an encoded CSR.